src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/CreateDossierComponentTest.kt

```diff
@@ -55,7 +47,8 @@ class CreateDossierComponentTest : ComponentTestBase() {
 
     val dossier = authenticatedClient.getDossier(recall.recallId)
 
-    assertThat(dossier.content, equalTo(expectedBase64NumberedPdf))
+//    File("test-results.pdf").writeBytes(Base64.getDecoder().decode(dossier.content))
```

Comment:
This file is more "mocked-response.pdf" than "test-results.pdf"?

Reply:
I will remove these comments that you added ;-)